src/bridge/tests/token_actions_test.cairo

```diff
@@ -179,8 +180,27 @@ fn reactivate_token_not_deactivated() {
         .write(usdc_address, TokenSettings { token_status: TokenStatus::Blocked, ..old_settings });
 
     mock.ownable.Ownable_owner.write(OWNER());
-    snf::cheat_caller_address_global(OWNER());
+    snf::start_cheat_caller_address_global(OWNER());
 
     mock.reactivate_token(usdc_address);
 }
 
+#[test]
+#[should_panic(expected: ('Already enrolled',))]
```

Comment:
I guess, Errors returned in asserts are felts and cannot be longer than 31 characters.
- So are you suggesting to make the error more generalistic to say 'Incorrect token Status'
- Or to check the current status and return a different error based on the current check(blocked, enrolled, deactivated etc). This might require something like match statement in the contract though, just for error

Reply:
`Incorrect token Status` sounds good